Description: Compile error fix with an additional include line
 This patch fixes a compile error by adding an #include <time.h>
Author: Roland Stigge <stigge@antcom.de>

--- swisswatch-0.6.orig/SwissWatch.c
+++ swisswatch-0.6/SwissWatch.c
@@ -19,6 +19,7 @@
 #include "Hand.h"
 #include "Mark.h"
 #include <math.h>
+#include <time.h>
 
 #if NeedFunctionPrototypes
 static void Initialize(Widget,Widget,ArgList,Cardinal*);

